### ISSUE TITLE
Crash when switching from 2D to 3D

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#5258]
Looking at this now in case it's an easy fix, if not we'll, @bagnell we need this for today's release, since it doesn't happen in 1.32

1. Run http://localhost:8080/Apps/Sandcastle/index.html?src=Hello%20World.html&label=Showcases
2. Switch to 2D.
3. Switch back to 3D and click to end morph early
4. Crash.

```
Uncaught DeveloperError: A PerspectiveFrustum or OrthographicFrustum is required in 3D and Columbus view
Error
    at new DeveloperError (http://localhost:8080/Source/Core/DeveloperError.js:44:19)
    at Camera.update (http://localhost:8080/Source/Scene/Camera.js:922:19)
    at http://localhost:8080/Source/Scene/SceneTransitioner.js:860:26
    at completeMorph (http://localhost:8080/Source/Scene/SceneTransitioner.js:271:17)
    at handleMouseDown (http://localhost:8080/Source/Core/ScreenSpaceEventHandler.js:167:13)
    at handlePointerDown (http://localhost:8080/Source/Core/ScreenSpaceEventHandler.js:609:13)
    at HTMLCanvasElement.listener (http://localhost:8080/Source/Core/ScreenSpaceEventHandler.js:68:13) (on line 922 of http://localhost:8080/Source/Scene/Camera.js)
```

The problem appears to be that completing am morph  early causes `camera.update` to be called while in 3D mode but with the incorrect frustum.

**Comment by mramato** [comment]
The problem appears to be the extra tween inserted into the morph sequence on line 442 of `SceneTransitioner.js` I believe this morph is there to go from orthographic to perspective before actually starting the morph.  This creates a 1 second window where calling `completeMorph` to complete early (which happens on mouse click) creates an invalid scene configuration because the orth-to-perspective morph hasn't completed yet.

The only thing that I could get to work would be to remove this "extra" tween completely, which resorted to morph working as intended, but I have no idea what the impact of this change is. Frankly this code is really hard to follow (I believe `SceneTransitioner` has been due for a rewrite forever). 

Hopefully the above will point to an obvious solution for @bagnell once he gets in.

**Comment by mramato** [comment]
@pjcozzi any thoughts here?  This is a pretty easy crash to recreate, so I think we need to fix this before release.  1.32 doesn't crash, but is broken differently with the same steps (wrong frustum in 3D)

**Comment by pjcozzi** [comment]
Let's take a fix from @bagnell if he is able to come up with it quickly; otherwise, if it is safe to remove the tween as you did or revert to something older, let's do it.

**Comment by mramato** [comment]
This appears to be similar to #5197, which was reported but never looked at.  I still hope we can fix in for 1.33.
